coalib/misc/Decorators.py

```diff
@@ -298,3 +298,9 @@ def decorated(*args, **kwargs):
         return function(*args, **kwargs)
 
     return decorated
+
+
+class classproperty(property):
+
```

Comment:
can you add a documentation comment with an example on how to use it as a doctest?